extension/solarwindsapmsettingsextension/extension.go

```diff
@@ -75,7 +90,91 @@ func (extension *solarwindsapmSettingsExtension) Shutdown(_ context.Context) err
 	return nil
 }
 
-func refresh(extension *solarwindsapmSettingsExtension) {
-	// Concrete implementation will be available in later PR
-	extension.logger.Info("refresh task")
+func refresh(extension *solarwindsapmSettingsExtension, filename string) {
+	extension.telemetrySettings.Logger.Info("time to refresh", zap.String("endpoint", extension.config.ClientConfig.Endpoint))
+	if hostname, err := os.Hostname(); err != nil {
+		extension.telemetrySettings.Logger.Error("unable to call os.Hostname()", zap.Error(err))
+	} else {
+		ctx, cancel := context.WithTimeout(context.Background(), grpcContextDeadline)
+		defer cancel()
+
+		request := &collectorpb.SettingsRequest{
+			ApiKey: extension.config.Key,
+			Identity: &collectorpb.HostID{
+				Hostname: hostname,
+			},
+			ClientVersion: "2",
```

Comment:
In `extension.Settings` there is a `build.Version` that you could use if you wanted to identify what collector version was built. 

I wonder if this should be set as part of the package coming in or use the provided collector version?